include/odp/api/spec/ipsec.h

```diff
@@ -191,6 +191,25 @@ typedef struct odp_ipsec_inbound_config_t {
 	 */
 	odp_proto_chksums_t chksums;
 
+	/** Post-IPsec reassembly configuration
+	 *
+	 *  This field provides global IPsec configuration parameters for
+	 *  fragment reassembly. The enable flag does not turn on reassembly
+	 *  but tells if reassembly may be enabled in SA parameters.
+	 *
+	 *  The enable flag may be set only if retain_outer is
+	 *  ODP_PROTO_LAYER_NONE.
+	 */
+	odp_reass_config_t reassembly;
+
+	/** Attempt reassembly after inbound IPsec processing in
+	 *  odp_ipsec_in_enq().
+	 */
+	odp_bool_t reass_async_mode;
+
+	/** Attempt reassembly after inline inbound IPsec processing. */
+	odp_bool_t reass_inline_mode;
```

Comment:
I think these should be named reass_async and reass_inline since these are not specific to the globally configured operation mode but the actual operation being done. We already made similar change in the description of these but missed the naming itself.

Reply:
Agreed. Missed this earlier. Will address in next version.